CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.8.0 (2019/01/02)
+
+* Support connection options :default_connection and :connection_builder when creating credentials that need to refresh OAuth tokens. This lets clients provide connection objects with custom settings, such as proxies, needed for the client environment.
+
```

Comment:
Do you want to mention the removal of the project_id warning in the release notes?

Reply:
Done.